rskj-core/src/main/java/co/rsk/peg/NoOpReturnException.java

```diff
@@ -0,0 +1,7 @@
+package co.rsk.peg;
```

Comment:
This can go inside peginstructions package since it's only thrown from PeginInstructionsProvider. And also inherit from PeginInstructionsException

Reply:
@guidohernan93 please move this Exception to the pegininstructions package